symphony/app/fbcnms-projects/hub/app/components/Main.js

```diff
@@ -106,9 +118,10 @@ export default () => {
     <ApplicationMain>
       <AppContextProvider>
         <Switch>
-          <Route path={relativeUrl('/services')} component={Main} />
+          <Route path={relativeUrl('/services')} component={Main(false)} />
           <Redirect exact from="/" to={relativeUrl('/hub')} />
           <Redirect exact from="/hub" to={relativeUrl('/services')} />
+          <Route path="/hub/workflows" component={Main(true)} />
```

Comment:
Something like: 
`<Route path={relativeUrl('/workflows')} component={...} />`
And placed under the `relativeUrl('/services')` tag